modules/apps/headless/headless-portal-instances/headless-portal-instances-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,204 @@
+components:
+    schemas:
+        PortalInstance:
+            # @review
+            properties:
+                active:
+                    readOnly: true
+                    type: boolean
+                companyId:
+                    format: int64
+                    type: integer
+                domain:
+                    description:
+                        domain used for email authentication.
+                    type: string
+                virtualhost:
```

Comment:
shouldn't be this virtualHost?

Reply:
No, we allow to define different the domain and virtual host. Take also into account that this is the internal virtual host. We should define the public virtual host in the web server.